home/web/client/components/CreateAccount/CreateAccount.js

```diff
@@ -149,26 +169,45 @@ export default class CreateAccount extends Web3Component {
         label='Create'
         className={ styles.submit }
         primary
-        disabled={ !this.state.isValid }
-        onTouchTap={ this.submit }
+        disabled={ !this.state.isValid || this.state.isCreatingAccount }
```

Comment:
I thought about doing that in a separate PR with a UI fix
and removal of validations